src/prog_algs/predictors/prediction.py

```diff
@@ -77,7 +77,7 @@ class UnweightedSamplesPrediction(Prediction, UserList):
             Data points where data[n] is a SimResult for sample n
     """
 
-    def __init__(self, times, data):
+    def __init__(self, times : list, data : UncertainData):
```

Comment:
```suggestion
    def __init__(self, times : list, data : list):
```

same as above- can you also update the doc comment